### ISSUE TITLE
Clear filter parameters when transitioning to another route

### ISSUE THREAD

**Issue by k-dauda** [BnitoBzh/ember-cli-debounce-query-params#3]

**Comment by BnitoBzh** [comment]
I think it is not a bug, you have to use the resetController method in your route in order to reset your controller query params.
https://emberjs.com/api/ember/3.0/classes/Route/methods/resetController?anchor=resetController

**Comment by k-dauda** [comment]
@BnitoBzh right, I am currently using a similar approach with the willTransition hook. However, given that you have to clear the actual query parameter as well as the debounce query parameter in the template, I think it would be helpful to have a method in the mixin to reset the query params to their initial value and clear the debounce params.
